### DISCUSSION TITLE
How can we add custom icon for custom python plugin chart

### DISCUSSION THREAD

**Discussion by botzill** [netdata/netdata#15050] [Q&A]
By default all custom plugins have this icon:

<img width="59" alt="Screenshot 2023-05-14 at 11 52 10" src="https://github.com/netdata/netdata/assets/4686040/a7d47648-eeba-4b1e-b524-965e574758a4">

Is it possible to add custom icon for python custom charts similar way we define:

```
self.order = ORDER
self.definitions = CHARTS
```

?

I do not see this options in the python code.

Thanks.

**Comment by ilyam8** [comment]
Hi, @botzill. See [Customize the standard dashboard info](https://learn.netdata.cloud/docs/visualizations/agent-dashboard/customize-the-standard-dashboard#customize-the-standard-dashboard-info), but this will work only for the local dashboard (not Cloud). And this dashboard will be deprecated in the future.

**Comment by botzill** [comment]
I see, I'm looking for the cloud one.
